### ISSUE TITLE
REFL012 IsDefined

### ISSUE THREAD

**Issue by brian-reichle** [DotNetAnalyzers/ReflectionAnalyzers#180]
The documentation for [REFL012](https://github.com/DotNetAnalyzers/ReflectionAnalyzers/blob/aa21e15b5232e051df2ccd4c47c153b12b8281e4/documentation/REFL012.md) would seem to suggest it should complain about the following, but I'm not getting anything. Has it not been implemented yet? did it break? or is it looking for a different pattern I haven't tried yet?

```csharp
if (typeof(Foo).GetCustomAttribute(typeof(BarAttribute)) != null)
{
}
```

I would expect it to suggest the following.

```csharp
if (typeof(Foo).IsDefined(typeof(BarAttribute)))
{
}
```

**Comment by brian-reichle** [comment]
Probably should have included `typeof(Foo).GetCustomAttribute<BarAttribute>() != null` :)
